src/components/Button.js

```diff
@@ -70,4 +72,22 @@ export const NavigationButton = styled(Button)`
   color: var(--background);
 `
 
+export const AddRecipeTitleToMealplanToggle = styled(Button)`
```

Comment:
Woho! Das ist mal ein Name! Aber mich verwirrt dass der Button was `Added` aber auch `Toggled` ?
Vielleicht `AddOrRemoveRecipeFromMealplan`? 
Dir fällt bestimmt was ein :)

Reply:
Wurde auch in der Umbenennungsaktion berücksichtigt. :)